app/models/subnet/ipv6.rb

```diff
@@ -9,6 +9,7 @@ class Subnet::Ipv6 < Subnet
   has_many :primary_hosts, :through => :primary_interfaces, :source => :host
 
   validate :validate_eui64_prefix_length, :if => Proc.new { |subnet| subnet.ipam == IPAM::MODES[:eui64]}
+  validates :mtu, :numericality => {:only_integer => true, :greater_than_or_equal_to => 1280, :less_than_or_equal_to => 65536}, :allow_nil => true
```

Comment:
`less_than_or_equal_to` should be 4,294,967,295 as per rfc 2675

Reply:
I was on the rfc8200 that don't specify maximum MTU, so I reported the IPv4 one. Your RFC talk about Jumbo datagram. I nerver saw that :). Fixed.